site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -186,6 +186,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.nodeSelector`                    | Controller [node labels][nodeSelector] for pod assignment                                       | `{}`                   |
 | `agones.controller.tolerations`                     | Controller [toleration][toleration] labels for pod assignment                                   | `[]`                   |
 | `agones.controller.affinity`                        | Controller [affinity][affinity] settings for pod assignment                                     | `{}`                   |
+| `agones.controller.annotations`                     | Controller [annotations][annotations] settings for pod assignment                               | `{}`                   |
```

Comment:
Please move this down to where we have the section "New Configuration Features:" - so they only get displayed on the next release :+1: 

Context: https://agones.dev/site/docs/contribute/#within-a-page

Reply:
Thank you. I fixed it with ab93fd6b5e51ebf65ce76e132a068b1e5584ff2e.